feature-pack/pom.xml

```diff
@@ -2892,6 +2892,15 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
60K	simpleclient/0.6.0/simpleclient-0.6.0.jar
8.0K	simpleclient_common/0.6.0/simpleclient_common-0.6.0.jar

Size is ok.
Which team own productization of those bits?
What's the sustaining/support plan for simpleclient ?